tests/starship/DEVNET.md

```diff
@@ -0,0 +1,82 @@
+## Mesh Security Devnet
+
+> NOTE: Host and smart contract address are subject to change on redeployment of the system
+
```

Comment:
IMHO it is better to use some different names like provider/ consumer instead that are not chain agnostic. The starship setup provides a meshd network and is not using osmosis/juno artifacts